src/components/CustomerAppeal/ReasonSelection/ReasonSelectionCard.tsx

```diff
@@ -52,9 +53,10 @@ export const ReasonSelectionCard: FunctionComponent<ReasonSelectionCard> = ({
             <ReasonSelectionHeader title={reasonSelectionHeader} />
           </View>
           <View style={styles.reasonPadding}>
-            {reasons.map((reason) => (
+            {reasons.map((reason, index) => (
               <ReasonItem
-                key={reason.description}
+                key={index}
+                category={reason.category}
```

Comment:
Why did we change from `description` to `category` here?

Also, are we adding the `index` for a reason?

Reply:
cause the sentry keeps throwing error since the `description` for `tt-lost-token` and `tt-lost-token-waive` are the same. 

I just read from the doc that `index` will be used by default anyway, so I think I should just remove it. Else, I'm thinking of using `reason.category` instead as the key.